packages/legacy/core/App/components/misc/CredentialCardPreview.tsx

```diff
@@ -0,0 +1,394 @@
+    },
+  })
+
+  const parseAttribute = (item: (Attribute & Predicate) | undefined) => {
+    let parsedItem = item
+    if (item && !item.value) {
+      parsedItem = pTypeToText(item, t, attributeTypes) as Attribute & Predicate
+    }
+    const parsedValue = formatIfDate(
+      attributeFormats?.[item?.name ?? ''],
+      parsedItem?.value ?? parsedItem?.pValue ?? null
+    )
+    return {
+      label: item?.label ?? item?.name ?? '',
+      value: item?.value ? parsedValue : `${parsedItem?.pType} ${parsedValue}`,
+    }
+  }
+
+  useEffect(() => {
+    const params = {
+      identifiers: { schemaId, credentialDefinitionId: credDefId },
+      attributes: [],
+      language: i18n.language,
+    }
+    OCABundleResolver.resolveAllBundles(params).then((bundle) => {
+      setOverlay({
+        ...overlay,
+        ...bundle,
+        brandingOverlay: bundle.brandingOverlay as BrandingOverlay,
+      })
+    })
+  }, [credDefId, schemaId])
+
+  const CredentialCardLogo: React.FC = () => {
+    return (
+      <View style={[styles.logoContainer, { elevation: elevated ? 5 : 0 }]}>
+        {overlay.brandingOverlay?.logo ? (
+          <Image
+            source={toImageSource(overlay.brandingOverlay?.logo)}
+            style={{
+              resizeMode: 'cover',
+              width: logoHeight,
+              height: logoHeight,
+              borderRadius: 8,
+            }}
+          />
+        ) : (
+          <Text
+            style={[
+              TextTheme.bold,
+              {
+                fontSize: 0.5 * logoHeight,
+                alignSelf: 'center',
+                color: '#000',
+              },
+            ]}
+          >
+            {(overlay.metaOverlay?.name ?? overlay.metaOverlay?.issuer ?? 'C')?.charAt(0).toUpperCase()}
+          </Text>
+        )}
+      </View>
+    )
+  }
+
+  const AttributeItem: React.FC<{ item: Attribute }> = ({ item }) => {
+    const label = (item.label || item.name)!
+    const ylabel = overlay.bundle?.labelOverlay?.attributeLabels[label] ?? startCase(label)
+    const [value, setValue] = useState(item.value)
+
+    useEffect(() => {
+      setValue(formatIfDate(item.format, value))
+    }, [])
+
+    return (
+      <View style={{ flexDirection: 'row' }}>
+        <Text
+          style={[
+            TextTheme.labelSubtitle,
+            styles.textContainer,
+            {
+              lineHeight: 19,
+              opacity: 0.8,
+            },
+          ]}
+          testID={testIdWithKey('AttributeName')}
+        >
+          {ylabel}
+        </Text>
+        <Text
+          style={[TextTheme.labelSubtitle, styles.textContainer, styles.valueText]}
+          testID={testIdWithKey('AttributeValue')}
+        >
+          {value}
+        </Text>
+      </View>
+    )
+  }
+
+  const PredicateItem: React.FC<{
+    item: Predicate
+  }> = ({ item }) => {
+    const { ColorPallet } = useTheme()
+    const label = (item.label || item.name)!
```

Comment:
I think we need another on of your `ylabels` here, looks like the oca branding isn't being applied to the predicate label in your screenshots